readme.md

```diff
@@ -15,18 +15,61 @@ Control the Neopixels directly attached to the board.
 ```js
 var Playground = require("playground-io");
 var five = require("johnny-five");
-var board = new five.Board({
-  io: new Playground({
-    port: "/dev/tty.usbmodem1411"
-  })
+var io = new Playground({
+  port: "/dev/tty.usbmodem1411"
 });
+var board = new five.Board({ io: io });
 board.on("ready", function() {
   var pixels = Array.from({ length: 10 }, function(_, index) {
     return new five.Led.RGB({
       controller: Playground.Pixel,
       pin: index
     });
-  });
+  }); 
+  
+  var led = new five.Led(13); 
+  
+  var buttonL = new five.Button('4', {
+    isPullup: true,
+    invert: true
+  }); 
```

Comment:
Turns out these params were unnecessary, can just be `new five.Button('4')`.

BTW, here is the full set of component initializations I've been using for testing: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/makerlab/BoardController.js#L225

I ended up needing to initialize the button _after_ the sound sensor on `A4`, otherwise the buttonL would not work (believe it would otherwise get set to the wrong mode).